package.json

```diff
@@ -34,6 +34,7 @@
     "babel-plugin-macros": "^3.1.0",
     "markdown-to-jsx": "^7.1.8",
     "next": "^12.3.3",
+    "next-i18next": "^13.0.3",
```

Comment:
Just wanted to check if this is used.

Reply:
Yup, the i18n config in next.config.js allows us to set a default locale and define our used locales. Default locale is needed for the middleware.